docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,114 @@
-# Running Tests using Mono Runtime
+# Running test suites using Mono
 
-## Running Runtime Tests
-We currently only support running tests against coreclr.  There are additional mono runtime tests in mono/mono, but they
-have not been moved over yet. Simply run the following command:
+Before running tests, [build Mono](../../building/mono/README.md) using the desired configuration.
 
+## Runtime Tests
+### Testing against Mono JIT:
```

Comment:
I really don't think you should specify the JIT here (I believe I took it out in my previous suggested edit) but instead Desktop, which also matching the other platforms mentioned below. You can test against any build configuration with this method; it would work perfectly fine for the interpreter, for example.

Reply:
It definitely won't work for AOT, which is supported. I know that the steps are different. But don't know the exact commands. LLVM is not supported yet, so can't predict that. Basically, it only works for JIT and interpreter. I will put interpreter there as well.